src/main/java/org/jenkinsci/account/Application.java

```diff
@@ -515,7 +519,7 @@ public HttpResponse doDoLogin(
             } finally {
                 context.close();
             }
-        } catch (AuthenticationException e) {
+        } catch (Exception e) {
```

Comment:
Bit worried that this exception doesn't seem to be logged, previously I assume it was being thrown up to the uncaught exception handler and getting logged there?

Because Exception is being caught there could be a number of infrastructure related reasons preventing log in, but with no log message here it could hide the problem